src/pages/studyView/StudyViewPage.tsx

```diff
@@ -47,16 +42,34 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
 
                 this.store.updateStoreFromURL(query);
             },
-            { fireImmediately: true }
+            {fireImmediately: true}
         );
     }
 
     private handleTabChange(id: string) {
-        this.props.routing.updateRoute({ tab: id });
+        this.props.routing.updateRoute({tab: id});
     }
 
-    content(){
+    @computed
+    get numberOfSamplesWithMutation() {
+        if (this.store.molecularProfileSampleCounts.isComplete &&
+            this.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples) {
```

Comment:
any reason for checking `numberOfMutationProfiledSamples`

Reply:
The endpoint does not always have numberOfMutationProfiledSamples